### PR TITLE
fix(react v7): Add missing HighContrast override to SplitButton

### LINKED ISSUE #16156: Button disabled background & text color properties do not take effect
### Environment Information

- **Package version(s)**: 7
- **Browser and OS versions**: Edge Chromium

### Please provide a reproduction of the bug in a codepen:

https://codepen.io/idigra/pen/GRjZXeb

#### Actual behavior:

When disabled, buttons background color is set to the default background color of disabled button.

#### Expected behavior:

When disabled, buttons background color is expected to be set to the color passed to loadTheme function in properties semanticColors.buttonBackgroundDisabled and semanticColors.primaryButtonBackgroundDisabled.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Excel

### Comments:

1. I found an issue that might be related. It doesn't seem like duplication as this issue is much simpler and deals with the classic case of disabled buttons (I just used the demo sample from the site with minimal modifications to demonstrate use of buttonBackgroundDisabled properties) - not like the other bug which is specific to high contrast mode:
https://github.com/microsoft/fluentui/issues/16125
2. I used the definition of semantic overrides as defined here: https://docs.microsoft.com/en-us/javascript/api/theme/isemanticcolors?view=office-ui-fabric-react-latest#primaryButtonBackgroundDisabled
3. The issue occurs also for primaryButtonTextDisabled.

### LINKED ISSUE #16339: On CommandBar with split and disabled buttons, enabled and disabled downward arrow are in same green color which creates confusion.
On CommandBar with split and disabled buttons in high contrast black mode, enabled and disabled downward arrow are in same green color which creates confusion.

### Environment Information

OS: Win 10
Browser:  Edge

### Describe the issue:

On CommandBar with split and disabled buttons in high contrast black mode, enabled and disabled downward arrow are in same green color which creates confusion.

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar

external_down_arrow

#### Actual behavior:
On CommandBar with split and disabled buttons, enabled and disabled downward arrow are in same green color which creates confusion.
#### Expected behavior:
On CommandBar with split and disabled buttons, enabled and disabled downward arrow should have different color to avoid confusion. Color may be white[for enabled one]

### PR DESCRIPTION
**Pull request by andrefcdias**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #16339
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Added a missing high contrast selector to the SplitButton styles.

###### Pre
image
###### Post
image

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts [hunk 1]
```diff
@@ -139,6 +139,11 @@ export const getStyles = memoizeFunction(
         marginTop: 0,
         marginRight: 0,
         marginBottom: 0,
+        [HighContrastSelector]: {
```
Comment (andrefcdias) [inline packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts:142]: Used generic `HighContrast` selector as the problem was also present in High Contrast White mode.

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/BaseButton-mount_base.html">897</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/BaseButton-mount.html">864</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Breadcrumb-mount_base.html">41859</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Breadcrumb-mount.html">42118</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Checkbox-mount_base.html">1505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Checkbox-mount.html">1527</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/CheckboxBase-mount_base.html">1287</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/CheckboxBase-mount.html">1257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/ChoiceGroup-mount_base.html">4864</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/ChoiceGroup-mount.html">4876</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/ComboBox-mount_base.html">927</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/ComboBox-mount.html">919</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/CommandBar-mount_base.html">7506</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/CommandBar-mount.html">7676</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/ContextualMenu-mount_base.html">13652</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/ContextualMenu-mount.html">13346</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DefaultButton-mount_base.html">1121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DefaultButton-mount.html">1115</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DetailsRow-mount_base.html">3488</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DetailsRow-mount.html">3604</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DetailsRowFast-mount_base.html">3591</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DetailsRowFast-mount.html">3509</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DetailsRowNoStyles-mount_base.html">3361</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DetailsRowNoStyles-mount.html">3493</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Dialog-mount_base.html">1446</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Dialog-mount.html">1483</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DocumentCardTitle-mount_base.html">1778</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/DocumentCardTitle-mount.html">1820</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Dropdown-mount_base.html">2477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Dropdown-mount.html">2464</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/FocusTrapZone-mount_base.html">1668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/FocusTrapZone-mount.html">1645</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/FocusZone-mount_base.html">1791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/FocusZone-mount.html">1798</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/IconButton-mount_base.html">1727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/IconButton-mount.html">1706</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Label-mount_base.html">331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Label-mount.html">331</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Layer-mount_base.html">1952</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Layer-mount.html">1981</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Link-mount_base.html">424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Link-mount.html">436</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/MenuButton-mount_base.html">1455</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/MenuButton-mount.html">1461</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/MessageBar-mount_base.html">2068</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/MessageBar-mount.html">2052</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Nav-mount_base.html">3153</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Nav-mount.html">3191</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/OverflowSet-mount_base.html">1379</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/OverflowSet-mount.html">1389</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Panel-mount_base.html">1394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Panel-mount.html">1405</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Persona-mount_base.html">839</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Persona-mount.html">793</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Pivot-mount_base.html">1403</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Pivot-mount.html">1430</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/PrimaryButton-mount_base.html">1245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/PrimaryButton-mount.html">1278</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Rating-mount_base.html">7316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Rating-mount.html">7426</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SearchBox-mount_base.html">1272</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SearchBox-mount.html">1227</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Shimmer-mount_base.html">2463</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Shimmer-mount.html">2488</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Slider-mount_base.html">1462</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Slider-mount.html">1472</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SpinButton-mount_base.html">4977</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SpinButton-mount.html">4935</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Spinner-mount_base.html">406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Spinner-mount.html">419</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SplitButton-mount_base.html">3072</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SplitButton-mount.html">3114</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Stack-mount_base.html">507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Stack-mount.html">494</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/StackWithIntrinsicChildren-mount_base.html">1507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/StackWithIntrinsicChildren-mount.html">1489</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/StackWithTextChildren-mount_base.html">4510</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/StackWithTextChildren-mount.html">4491</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SwatchColorPicker-mount_base.html">9920</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/SwatchColorPicker-mount.html">10033</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/TagPicker-mount_base.html">2697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/TagPicker-mount.html">2700</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/TeachingBubble-mount_base.html">50200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/TeachingBubble-mount.html">50413</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Text-mount_base.html">441</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Text-mount.html">418</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/TextField-mount_base.html">1342</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/TextField-mount.html">1330</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Toggle-mount_base.html">818</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/Toggle-mount.html">814</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/button-mount_base.html">94</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pr-deploy-site/refs/pull/16635/merge/perf-test/button-mount.html">106</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-examples-2021-01-22-12-35-03-hotell-cherry-pick-16439.json [hunk 2]
```diff
@@ -1,8 +0,0 @@
-{
-  "type": "patch",
```
Comment (ecraig12345) [inline change/@fluentui-react-examples-2021-01-22-12-35-03-hotell-cherry-pick-16439.json:2]: Was this intentional?

Comment (andrefcdias) [reply change/@fluentui-react-examples-2021-01-22-12-35-03-hotell-cherry-pick-16439.json:2]: Definitely not, always fighting against the changes 😄 Will fix.

**Review by khmakoto — APPROVED** [review]

**Comment by v-ajrai** [comment]
Can you let us know which version of fabric UI has this changes incorporated so that we could use it.

**Comment by ecraig12345** [comment]
@v-ajrai It's in a comment just above yours.
> 🎉`office-ui-fabric-react@v7.157.2` has been released which incorporates this pull request.🎉

**Pull request merged**